client/cz-component-dash.html

```diff
@@ -104,57 +104,74 @@
           type: 'Object',
           value: function() { return []; },
         },
-        component: {
+        updateSLO: {
           type: 'Object',
-          value: function() { return {}; },
+          value: function() { return null; },
```

Comment:
why this change? Also you don't need a function to initialize objects to null, just use value: null.